_content/essays/love-unknowing-and-female-filth_amy_paris_langenberg.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Love, Unknowing, and Female Filth: The Buddhist Discourse of Birth as a Vector of Social Change for Monastic Women in Premodern South Asia"
+authors:
+  - "Amy Paris Langenberg"
+external_url: "https://www.degruyter.com/document/doi/10.1515/9783110674088-012/pdf"
+source_url: "https://www.degruyter.com/document/doi/10.1515/9783110674088-012/html"
+drive_links:
+  - "https://drive.google.com/file/d/1vh9BaxP8ORYmgD1FBIRTvjvWAJe2IQJe/view?usp=share_link"
+formats: [pdf]
+course: nature
+tags:
+  - gender
+  - vinaya
+  - nun
```

Comment:
The tag is `nuns` plural. And let's maybe also add the `bhikkhuni` tag.

PS: There was a work ("av/first-female-geshe_kelsang") tagged with `nun` but that's a typo which I'll fix shortly 😅 Sorry about that!